api/src/main/java/org/pmiops/workbench/db/jdbc/ReportingQueryServiceImpl.java

```diff
@@ -255,6 +256,12 @@ public long getQueryBatchSize() {
                 + "      WHERE uat.access_status = 1 " // ENABLED
                 + "      GROUP BY u.user_id"
                 + "  ) as t ON t.user_id = u.user_id "
+                // temporary solution to RW-6566: retrieve first_enabled from user_access_tier
```

Comment:
semantically: this is truthful and complete.  `user_access_tier.first_enabled` contains the first-registration-time for new users and it was also backfilled with `user.first_registration_completion_time` for existing users.